pkg/kubelet/prober/prober.go

```diff
@@ -195,6 +198,21 @@ func (pb *prober) runProbe(probeType probeType, p *v1.Probe, pod *v1.Pod, status
 		klog.V(4).InfoS("TCP-Probe Host", "host", host, "port", port, "timeout", timeout)
 		return pb.tcp.Probe(host, port, timeout)
 	}
+
```

Comment:
This needs to be implemented behind the `GRPCContainerProbe` feature flag, per the KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2727-grpc-probe#feature-enablement-and-rollback